tasks/task179_participant_extraction.json

```diff
@@ -0,0 +1,2577 @@
+{
+    "Definition": "In medical studies, treatments are tested within a group of study participants. You will be given a sentence of a study report in which your task is to list the tokens that  gives information about the participants of the study. If no information about the participants is mentioned, just output \"not found\". Relevant information about participants include: gender, medical conditions, location, number of people participating. Do not contain participant mentions without relevant information.",
```

Comment:
"tokens that  gives": typo

Reply:
- "list the tokens": if the answer is always a sub-phrase/sub-string from the given sentence, it might be easier to use any of these keywords (instead of "tokens" which is less common among lay speakers of English).

Reply:
Combine the following two sentences? 
> Relevant information about participants include: gender, medical conditions, location, number of people participating. Do not contain participant mentions without relevant information. 

For example:  
> Mentions of participants should include relevant information such as gender, ....